src/support/stat.c

```diff
@@ -1457,6 +1457,8 @@ static const char *const __stats_connection_desc[] = {
   "block-manager: number of times the file was remapped because it changed size via fallocate or "
   "truncate",
   "block-manager: number of times the region was remapped via write",
+  "block-manager: time spent on the most recent linear walk of extents during first-fit "
+  "allocation(usecs)",
```

Comment:
NP: more readable to move the unit of time up. i.e. "time spent (usecs) on the most..."